bentoml/server/open_api.py

```diff
@@ -61,9 +61,34 @@ def get_open_api_spec_json(bento_service):
                 "in JSON format and must contain a `request_id` field, which can be "
                 "obtained from any BentoService API response header",
                 responses=default_response,
-            )
+            ),
+            post=OrderedDict(
+                tags=["infra"],
+                description="Predictions feedback endpoint. Expecting feedback request "
+                "in JSON format and must contain a `request_id` field, which can be "
+                "obtained from any BentoService API response header",
```

Comment:
This is great.  We can be more descriptive to let users know what feedback endpoint does.  

Maybe something along the line of ```Provide feedback to prediction results from BentoService.  Expecting feedback request payload in JSON format and requires `request_id` field, which can be obtained from any BentoService prediction response's header```

Reply:
Done!